### DISCUSSION TITLE
Stereo Camera View (Ar in Stereo Display)

### DISCUSSION THREAD

**Discussion by div-t** [AR-js-org/AR.js#397] [Q&A]
**Do you want to request a *feature* or report a *bug*?**
I have a question?

**What is the current behavior?**
My software should show AR-content. I can hold my phone in my hand and see it and this works fine. But I want to use my phone with a cardboard, so the idea is that I open the page and put the phone in a cardboard and then see my AR-Software in Stereo Display. I wasn't able to find anything about that in your documentation.

**If the current behavior is a bug, please provide the steps to reproduce.**

**Please mention other relevant information such as the browser version, Operating System and Device Name**
I am testing it on Iphone SE2 with Safari and iOS 14.8.1
 
**What is the expected behavior?**
The idea is that there is a button to switch into a cardboard stereo display mode and use the software handsfree in a Google Cardboard. Is there any possibility to do that yet.

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Comment by kalwalt** [comment]
Hi @div-t what branch have you tested? Have you some code to test? I have not an iOS device so i can't really help but maybe other users can discuss with you.
